client/src/components/ThirdParty.tsx

```diff
@@ -54,9 +54,19 @@ export const MicrosoftLogin = () => {
 };
 
 export const GitHubLogin = () => {
+  const CLIENT_ID = process.env.GITHUB_CLIENT_ID;
```

Comment:
To get `environment variables` in React to work follow this guide:
https://stackoverflow.com/questions/49108136/importing-env-variable-react-front-end